include/kokkos-utils/callbacks/EventQueueMatcher.hpp

```diff
@@ -8,19 +8,14 @@ namespace Kokkos::utils::callbacks
 {
 
 //! Match an event whose @c dev_id is the same as the one of @ref exec.
-template <Matcher MatcherType, Kokkos::utils::concepts::ExecutionSpace Exec>
+template <Kokkos::utils::concepts::ExecutionSpace Exec>
```

Comment:
Indeed, let's try to use some `And<MatcherType, EventQueueMatcher>` instead.

Reply:
Done. Thanks to:
- https://github.com/uliegecsm/kokkos-utils/pull/78